examples/external_decorators/README.md

```diff
@@ -0,0 +1,21 @@
+# External decorators demo
+
+This example is in response to this [GitHub issue](https://github.com/DAGWorks-Inc/hamilton/issues/50). However, it showcases more general mechanisms of how to parse "Hamilton  modules" and dynamically alter source code.
+
+# Objective
+Use an existing Python module that has no Python dependencies and decorated it with Hamilton functionalities without altering the source file.
+
+# Content
+The script run.py will:
```

Comment:
OK, so we have two different approaches here, and they're a little confusing. Let's highlight the two use-cases separately, and make `run.py` do both of them distinctly.